WickedEngine/Utility/FAudio/CMakeLists.txt

```diff
@@ -1,19 +1,20 @@
 # CMake Project for FAudio
 # Written by @NeroBurner
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.5...3.10)
```

Comment:
why the `...3.10` ?
I thought it came from upstream, but you added this yourself instead. What is the reason? (and maybe add a comment for it for future developers)

Reply:
It get rids of a CMake compatibility warning (supporting pre-3.10 behaviour is deprecated. This syntax says it's compatible with 3.10) I can't remember why we changed it to 3.5 in the first place, though; but since our main CMakeLists depends on 3.12 anyway, it's more of a technicality. Maybe we should just change it to 3.12 as well and be done with it?

Reply:
Both are fine, whatever you feel like it's better